docs/kr.tree

```diff
@@ -209,6 +209,10 @@
             <toc-element id="native-faq.md"/>
             <toc-element toc-title="Intro to Kotlin/Native – hands-on" href="https://play.kotlinlang.org/hands-on/Introduction%20to%20Kotlin%20Native/"/>
         </toc-element>
+
+        <toc-element toc-title="Scripting">
+            <toc-element id="custom-script-deps-tutorial.md"/>
+        </toc-element>
```

Comment:
Scripting is not a platform though.
But it's hard to put it to another section...
We should revise this section to understand where to put Scripting section.

Reply:
Indeed, this is a difficult question. It's not a platform, but it seems close to available platforms semantically. Field of application or smth like this.
Another option was adding a new top-level section, but it seems awkward to create it for a single top-level tutorial with a limited audience. Maybe we can consider this when I complete at least three planned pages: this tutorial, code prototyping, and main.kts scripting.